official/resnet/resnet_run_loop.py

```diff
@@ -26,21 +26,27 @@
 import argparse
 import os
 
-import tensorflow as tf  # pylint: disable=g-bad-import-order
+# pylint: disable=g-bad-import-order
+import tensorflow as tf
+from tensorflow.contrib.distribute.python import mirrored_strategy
+from tensorflow.contrib.distribute.python import one_device_strategy
+from tensorflow.python.training import distribute as distribute_lib
 
 from official.resnet import resnet_model
 from official.utils.arg_parsers import parsers
 from official.utils.export import export
 from official.utils.logs import hooks_helper
 from official.utils.logs import logger
+# pylint: enable=g-bad-import-order
```

Comment:
Once we move to top-level imports as @guptapriya notes above, we can just switch back to the single-line import order.